var/spack/repos/builtin/packages/isaac/package.py

```diff
@@ -36,6 +37,11 @@ class Isaac(CMakePackage):
     depends_on('jansson', type='link')
     depends_on('boost@1.56.0:', type='link')
     depends_on('boost@1.65.1:', type='link', when='^cuda@9:')
+
+    # TODO: replace this with an explicit list of components of Boost,
+    # for instance depends_on('boost +filesystem')
+    # See https://github.com/spack/spack/pull/22303 for reference
+    depends_on(Boost.with_default_variants, type='link')
```

Comment:
Since it depends only on header-only Boost libs, is this the right dependency spec to keep?
```suggestion
```